### PR TITLE
Initial edits

### PR DESCRIPTION
**Pull request by younglee327**
# Fixes:
- Mobile menu responsiveness
- About section height adjusted on mobile
- Made @ Penn section background picture darkened to improve text readability
- Contact section centred on tablet view
- Bug report/suggestions links go to new tab to Google Forms
- Optimized all images losslessly
- All pictures unskewed through CSS fit
- Footer no longer overflows on mobile
- Menu no longer overflows on tablet size
- Scrolling animation to sections upon menu selection

### CODE REVIEW

**Comment by adelq** [comment]
Hey Young, I won't have time for a full review right now, just giving initial comments right now. Looks like a lot of work done!

File: _includes/footer.html [hunk 1]
```diff
@@ -3,16 +3,22 @@
...
 
-<!-- Bootstrap core JavaScript
+<!-- Bootstrap and jQuery core JavaScript
      ================================================== -->
 <!-- Placed at the end of the document so the pages load faster -->
-<script async src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
-<script src="/js/bootstrap.min.js"></script>
+
+<!-- <script async src="https://code.jquery.com/jquery-1.10.2.min.js"></script> -->
+<script src="https://code.jquery.com/jquery-3.1.1.min.js" integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8=" crossorigin="anonymous"></script>
```
Comment (adelq) [inline _includes/footer.html]: Why not async anymore?

File: _includes/header.html [hunk 2]
```diff
@@ -57,22 +58,23 @@
...
+            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target=".navbar-collapse" aria-expanded="false">
+              <span class="sr-only">Toggle navigation</span>
               <span class="icon-bar"></span>
               <span class="icon-bar"></span>
               <span class="icon-bar"></span>
             </button>
           </div>
           <div class="collapse navbar-collapse">
             <ul class="nav navbar-nav">
-              <li {% if page.url == '/index.html' %}class="active"{% endif %}><a href="/#banner">Home</a></li>
```
Comment (adelq) [inline _includes/header.html]: This part is necessary so the tab actually activates, definitely needs to be reverted

File: _includes/header.html [hunk 3]
```diff
@@ -57,22 +58,23 @@
...
+              <li><a href="/#banner">Home</a></li>
               <li><a href="/#projects">Projects</a></li>
               <li><a href="/#about">About</a></li>
               <li><a href="/#team">Team</a></li>
-              <li {% if current[1] == 'alumni' %}class="active"{% endif %}><a href="/alumni">Alumni</a></li>
-              <li><a href="http://blog.pennlabs.org/">Blog</a></li>
+              <li><a href="/alumni">Alumni</a></li>
+              <!-- <li><a href="http://blog.pennlabs.org/" target="_blank">Blog</a></li> -->
```
Comment (adelq) [inline _includes/header.html]: Same here about tab activation, also why did you remove the blog link?

File: css/style.css [hunk 4]
```diff
@@ -350,34 +357,111 @@ form ul, ol{
...
+  #footer-alt-text {
+    display: inline;
+  }
+
+  .help-form {
+    top: 50px;
+    text-align: center;
+  }
+
+}
```
Comment (adelq) [inline css/style.css]: Shouldn't this also be below with the rest of the media queries?

File: index.html [hunk 5]
```diff
@@ -64,11 +64,9 @@
...
-    <p>While Penn already had an API for its registrar, Labs worked with the administration to open up the API for developer use. With an easy to use python wrapper, the API allows developers to access up-to-date registrar information, including course schedules, registration status and locations. <a
-                                                                                                                                                                                                                                                                                                              href="https://secure.www.upenn.edu/computing/da/webloginportal/eforms/index.html?content=kew/EDocLite?edlName=registrarApiAccessForm&userAction=initiate">Request access to the api here.</a></p>
-    <p></p>
-  </div>
+    <p>While Penn already had an API for its registrar, Labs worked with the administration to open up the API for developer use. With an easy to use python wrapper, the API allows developers to access up-to-date registrar information, including course schedules, registration status and locations.</p>
```
Comment (adelq) [inline index.html]: Why did you remove the API request link? I don't think it's linked anywhere else.

File: js/scripts.js [hunk 6]
```diff
@@ -1,62 +1,26 @@
...
-   var id = cur && cur.length ? cur[0].id : "";
-   
-   if (lastId !== id) {
-       lastId = id;
-       // Set/remove active class
-       menuItems
-         .parent().removeClass("active")
-         .end().filter("[href=#"+id+"]").parent().addClass("active");
-   }                   
-});
```
Comment (adelq) [inline js/scripts.js]: Are you also getting rid of the active tab during scrolling?

Comment (younglee327) [inline js/scripts.js]: I redid the scrolling JS because the nav is not sticky for this website, which would mean that even if the right section became active while scrolling, the user wouldn't be able to see it. Should I revert it anyway?

File: index.html (hunk 5 above)
Comment (younglee327) [inline index.html]: Oh oops! This was a mistake. Sublime had some weird glitch where it displayed like 50 extra blank lines (even though this didn't show up anywhere else), so I tried deleting it but it seems to have deleted that link too! I'll revert this asap

File: _includes/footer.html (hunk 1 above)
Comment (younglee327) [inline _includes/footer.html]: I read this article: https://css-tricks.com/async-attribute-scripts-bottom/

I'm not too sure how valid this article is, so please let me know if I should revert it!

Comment (adelq) [inline _includes/footer.html]: :+1:

File: _includes/header.html (hunk 3 above)
Comment (adelq) [inline _includes/header.html]: Well...I guess that needs fixing. Open bug reports when you find something broken!
https://github.com/pennlabs/blog/issues/1

File: css/style.css (hunk 4 above)
Comment (adelq) [inline css/style.css]: :+1:

File: index.html (hunk 5 above)
Comment (adelq) [inline index.html]: :+1:

File: js/scripts.js (hunk 6 above)
Comment (adelq) [inline js/scripts.js]: Oh...it used to be a sticky header, interesting, don't remember when that stopped working. It could have been me honestly, I'm not a fan of sticky headers, but lots of deleted lines without explanation in the commit messages is confusing :)

Your choice whether you want to make the header sticky again and keep the code or get rid of it.

File: _includes/header.html (hunk 2 above)
Comment (adelq) [inline _includes/header.html]: Ah, good call, yeah that's a bug, it should just be `if page.url == '/'`. Having an active tab would be nice though since the alumni page is separate, and the nav is visible there.

File: _includes/footer.html (hunk 1 above)
Comment (adelq) [inline _includes/footer.html]: Also, you can get rid of the old script instead of adding it as a comment.

File: _includes/header.html (hunk 3 above)
Comment (adelq) [inline _includes/header.html]: Blog is back up!

File: _includes/footer.html (hunk 1 above)
Comment (younglee327) [inline _includes/footer.html]: Got it!

File: _includes/header.html (hunk 2 above)
Comment (younglee327) [inline _includes/header.html]: Reverted!

File: _includes/header.html (hunk 3 above)
Comment (younglee327) [inline _includes/header.html]: Uncommented!

**Comment by younglee327** [comment]
All conflicts have been resolved with the master branch! Also, I think we should keep the header not sticky, so all issues with this branch should be resolved.

**Comment by adelq** [comment]
LGTM
